.github/pull_request_template.md

```diff
@@ -1,22 +1,15 @@
-**Notes for Reviewers**
+<!-- Thank you for contributing to GDSCITM! This is a template that makes it easier to review your changes: -->
 
-<!--tag the issue id  -->
-This PR fixes #
```

Comment:
@WebSnke thanks for making changes but we require this line of it will be helpful if you will not remove it

Reply:
I didn’t remove it. It is in line 15 now.

Reply:
Thanks for confirming